src/webpack/index.ts

```diff
@@ -45,10 +49,10 @@ function buildManifest(compiler: Compiler, compilation: Compilation) {
 
         if (
           currentModule.rawRequest &&
```

Comment:
Do we need the `currentModule.rawRequest` check now that `name` is used?

Reply:
Perhaps we do, there's a bit more logic related to this other than just the conditional here. There are two ways that `currentModule` can be defined, and based on either of those code paths we are checking if a `rawRequest` prop exists.